nvml/datadog_checks/nvml/nvml.py

```diff
@@ -73,11 +73,30 @@ class NvmlCheck(AgentCheck):
     """Lock for the object known_tags."""
     _thread = None
     """Daemon thread updating k8s tag information in the background."""
+    should_run = False
+    """Whether libnvml can be found and the check can thus run."""
+
+    def __init__(self, name, init_config, instances):
+        super(NvmlCheck, self).__init__(name, init_config, instances)
+        # self.N = pynvml
+        if self.is_nvml_library_available():
```

Comment:
I think it should log something if its not available (error or debug). On machines where a GPU is not available the check should not be enabled in the first place.

Reply:
Looks like this is part of is_nvml_library_available now